src/map/vending.cpp

```diff
@@ -67,7 +67,7 @@ void vending_closevending(map_session_data* sd)
 
 		sd->state.vending = false;
 		sd->vender_id = 0;
-		clif_closevendingboard(&sd->bl, 0);
+		clif_closevendingboard( sd->bl );
```

Comment:
Default values seem useless for 1 occurrence
```suggestion
		clif_closevendingboard( sd->bl, AREA_WOS, nullptr );
```